lib/cryptoexchange/exchanges/exnce/services/market.rb

```diff
@@ -0,0 +1,46 @@
+module Cryptoexchange::Exchanges
+  module Exnce
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super(ticker_url)
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Exnce::Market::API_URL}/ticker"
+        end
+
+        def adapt_all(output)
+          timestamp = output['date']
+          output['data'].map do |ticker|
+            next unless ticker['pair']
+            adapt(ticker, timestamp)
+          end.compact
+        end
+
+        def adapt(output, timestamp)
```

Comment:
can support 24 hours `high` and `low` as well by adding:
```
          ticker.high      = NumericHelper.to_d(output['high'])
          ticker.low       = NumericHelper.to_d(output['low'])
```

Reply:
its supports high and low let me add it quickly.

Reply:
i added ticker high and low parameters.